### ISSUE TITLE
Replace file() with open() in twisted/scripts

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8522]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8522|
|Type|enhancement|
|Created|2016-06-28 06:17:44Z|
|Branch|https://github.com/twisted/twisted/tree/8522-rodrigc-file|

In Python 3, file() is gone (see https://docs.python.org/3/whatsnew/3.0.html#builtins ), and needs to be replaced with open()
which works on Python 2 and 3

<details><summary>Searchable metadata</summary>

```
trac-id__8522 8522
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8522_rodrigc_file 8522-rodrigc-file
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467094664572026 1467094664572026
changetime__1468044991813942 1468044991813942
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/246

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

There are some Windows buildbot failures here but they seem to be intermittent and unrelated. Merge when ready.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

Needs more coverage in order to commit

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset 20df2cf556dce3818e7e71047cefea5b4aac4862
```
#!CommitTicketReference repository="" revision="20df2cf556dce3818e7e71047cefea5b4aac4862"
Merge pull request #246 from twisted/8522-rodrigc-file

Author: rodrigc
Reviewer: Lukasa
Fixes: #8522
Replace file() with open() in twisted/scripts
```
